tests/e2e/specs/out_of_game/lobby.spec.js

```diff
@@ -1,12 +1,12 @@
-const validEmail = 'myCustomEmail@gmail.com';
+const validUsername = 'fooBarTheBazBuzz';
 const validPassword = 'passwordLongerThanEight';
-const opponentEmail = 'yourMortalEnemy@cia.gov';
+const opponentUsername = 'definitelyNotTheGovernment6969';
```

Comment:
This seems a reasonable time to consolidate the validUsername, invalidUsername, and opponentUserName into a centralized place for consistent imports